internal/unix/types_other.go

```diff
@@ -27,14 +27,14 @@ const (
 	EACCES = syscall.Errno(0)
 	EILSEQ = syscall.Errno(0)
 
-	BPF_F_NO_PREALLOC        = 0
+	BPF_F_NO_PREALLOC        = 1 << 0
```

Comment:
Why do we need to touch those? I always thought they were set to 0 on purpose.

Reply:
The Go compiler rejects the switch statement due to duplicate values I think.

Reply:
Yes this is the issue, that it's trying to fix